<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оптика &quot;Люкс&quot;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Визиты</string>
     </attribute>
     <widget class="QTableView" name="client_tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>331</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="visit_tableView">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>321</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_client_pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="accept_client_pushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Подтвердить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_client_pushButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_visit_pushButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="change_visit_pushButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_visit_pushButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="client_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>54</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Клиент</string>
      </property>
     </widget>
     <widget class="QLabel" name="visit_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>55</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата визита</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="search_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="search_checkBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Вкл.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Дисконт</string>
     </attribute>
     <widget class="QTableView" name="discount_tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>651</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_discount_pushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="accept_discount_pushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>280</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Подтвердить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_discount_pushButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="discount_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>3</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер карты</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cansel_pushButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Справочник</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>651</width>
        <height>241</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Врач</string>
       </attribute>
       <widget class="QTableView" name="doctors_tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>621</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Причина обращения</string>
       </attribute>
       <widget class="QTableView" name="reason_tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>621</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Бренд</string>
       </attribute>
       <widget class="QTableView" name="brend_tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>621</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Наименование товара</string>
       </attribute>
       <widget class="QTableView" name="goods_tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>621</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Средство ухода</string>
       </attribute>
       <widget class="QTableView" name="agent_care_tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>621</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="add_directory_pushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="accept_directory_pushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>280</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Подтвердить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_directory_pushButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cansel_directory_pushButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action">
   <property name="text">
    <string>Настройки соединения</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
